### PR TITLE
Cover form email component in checkout page with jasmine test

### PR DESCRIPTION
**Pull request by mrtuvn**
Reopen closed my PR. This PR aims to apply cover test jasmine for js component 
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Checkout/view/frontend/web/js/view/form/element/email.js

### Related Pull Requests

https://github.com/magento/magento2/pull/27691

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by mrtuvn** [comment]
@magento run all tests

File: dev/tests/js/jasmine/tests/app/code/Magento/Checkout/frontend/js/view/form/element/email.test.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            'Magento_Checkout/js/model/full-screen-loader': jasmine.createSpy(),
+            'mage/validation': jasmine.createSpy()
+        },
+        obj;
+
+        beforeEach(function (done) {
+            window.checkoutConfig = {
+                quoteData: {
+                    /* jscs:disable requireCamelCaseOrUpperCaseIdentifiers */
+                    entity_Id: 1
```
Comment (ihor-sviziev) [inline dev/tests/js/jasmine/tests/app/code/Magento/Checkout/frontend/js/view/form/element/email.test.js]: I think it should be like this:
```suggestion
                    entity_id: 1
```
Is it mistake?

**Comment by mrtuvn** [comment]
@magento run all tests

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: dev/tests/js/jasmine/tests/app/code/Magento/Checkout/frontend/js/view/form/element/email.test.js (hunk 1 above)
Comment (mrtuvn) [reply dev/tests/js/jasmine/tests/app/code/Magento/Checkout/frontend/js/view/form/element/email.test.js]: https://github.com/magento/magento2/blob/2.4-develop/dev/tests/js/jasmine/tests/app/code/Magento/Vault/view/frontend/web/js/view/payment/method-renderer/vault.test.js#L48
This file still use entity_Id

Comment (mrtuvn) [reply dev/tests/js/jasmine/tests/app/code/Magento/Checkout/frontend/js/view/form/element/email.test.js]: But i think use entity_id is more make sense than entity_Id

**Comment by mrtuvn** [comment]
@magento run tests

**Comment by mrtuvn** [comment]
@magento run tests

**Comment by mrtuvn** [comment]
@magento run all tests

**Comment by mrtuvn** [comment]
@omiroshnichenko can you review this

File: dev/tests/js/jasmine/tests/app/code/Magento/Checkout/frontend/js/view/form/element/email.test.js (hunk 1 above)
Comment (ihor-sviziev) [reply dev/tests/js/jasmine/tests/app/code/Magento/Checkout/frontend/js/view/form/element/email.test.js]: Could you check what we have on the frontend? If it's entity_Id - let's leave it test as is, if it's entity_id - let's update the test

**Comment by ihor-sviziev** [comment]
@magento run Integration Tests

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-7913](https://jira.corp.magento.com/browse/ENGCOM-7913) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

**Pull request merged**